docs/getting-started.md

```diff
@@ -0,0 +1,184 @@
+- [Staging](https://bvdp-saturn-staging.appspot.com/)
+
+For [production](https://app.terra.bio/), you will need to register using a
+`firecloud.org` email. In order to get an account, you must become suitable,
+which requires following [these steps](https://docs.google.com/document/d/1DRftlTe-9Q4H-R0jxanVojvyNn1IzbdIOhNKiIj9IpI/edit?usp=sharing).
+
+Ask a member of the team to add you to the admins group for each of these
+environments.
+
+## 5. Install Homebrew
+
+[Homebrew](https://brew.sh/) is a [package manager](https://en.wikipedia.org/wiki/Package_manager)
+which enables the installation of software using a single, convenient command
+line interface. To automatically install development tools necessary for the
+team, a [Brewfile](https://github.com/Homebrew/homebrew-bundle) is used:
+
+```
+/usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+curl -LO https://raw.githubusercontent.com/DataBiosphere/terra-data-catalog/main/docs/Brewfile
+brew bundle --no-lock install
+```
+
+1. [Git](https://git-scm.com/) is a version control tool for tracking changes in
+projects and code.
+2. [git-secrets](https://github.com/awslabs/git-secrets) prevents developers
+from committing passwords and secrets to git.
+3. [jq](https://stedolan.github.io/jq/) is a command line JSON processing tool.
+4. [Docker](https://www.docker.com/) is a tool to deliver software in packages
+called containers. Docker for MacOS also includes [Kubernetes](https://kubernetes.io/),
+which deploys groups of containers together in clusters.
+5. [Vault](https://www.vaultproject.io/) is an encrypted database used to store
+many of the team's secrets such as keys and passwords.
+6. [Google Cloud SDK](https://cloud.google.com/sdk) is a command-line interface
+to Google Cloud services. Once it is installed, you'll need to allow auth access
+and configure Docker to connect to the appropriate Google Cloud endpoint when
+necessary, which is done with the configuration below.
+7. [IntelliJ IDEA](https://www.jetbrains.com/idea/) is an integrated development
+environment (IDE) for Java. There are two versions available: **Ultimate** (paid)
+and **Community** (open-source). We recommend the Ultimate Edition to Broad
+employees for its database navigation capabilities. Alternatively, the Community
+Edition has all the features needed for development, and this version can be
+installed by switching `intellij-idea` with `intellij-idea-ce` in the Brewfile.
+8. [Temurin](https://adoptium.net/) is the code name for the free and
+open-source version of Java 17 provided by the Eclipse Adoptium Working Group.
+
+Unfortunately, some manual configuration is also necessary:
+
+```
+# configure vault
+export VAULT_ADDR=https://clotho.broadinstitute.org:8200
+
+# launch docker desktop - this installs docker in /usr/local/bin
+open -a docker
+
+# configure google-cloud-sdk
+gcloud auth login
+gcloud auth application-default login
+gcloud auth configure-docker
```

Comment:
I think these are only needed if you're working on debugging k8s or helm. We don't use BQ and our app itself doesn't auth with google directly (I think). I guess they'd be useful if you want to proxy to our databases for debugging an issue in dev or prod.

Reply:
I think this is useful for debugging later on, so I'll keep these in for now.